<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeWidget</class>
 <widget class="QWidget" name="EmployeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeWidget</string>
  </property>
  <widget class="QLineEdit" name="contactLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>108</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="roleLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>108</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="hoursSpin">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="performanceSpin">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
